src/app/shared/button/button.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss'],
+})
+export class ButtonComponent {
+  @Input() buttonText = ' 2';
```

Comment:
why there is a space inside?

Reply:
I make mistake and forgot remove test data.